--- a/getopt.h	2025-10-07 02:04:49.129876712 +0000
+++ b/getopt.h 	2025-10-07 02:05:34.320876712 +0000
@@ -102,7 +102,7 @@
    errors, only prototype getopt for the GNU C library.  */
 extern int getopt(int argc, char *const *argv, const char *shortopts);
 #else  /* not __GNU_LIBRARY__ */
-extern int getopt();
+extern int getopt (int, char *const *, const char *);
 #endif /* __GNU_LIBRARY__ */
 extern int getopt_long(int argc, char *const *argv, const char *shortopts, const struct option *longopts, int *longind);
 extern int getopt_long_only(int argc, char *const *argv, const char *shortopts, const struct option *longopts,
